src/index.html

```diff
@@ -15,13 +15,392 @@
+              <a
+                class="header__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="grid">
+          <div
+            class="card"
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="Product card"
+              class="card__image"
+            />
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews-text">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-label">Price:</p>
+              <p class="card__price-value">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              BUY
+            </a>
+          </div>
+          <div
```

Comment:
Add the blank line between the cards
```suggestion
          </div>
          
          <div
```